.eslintrc.js

```diff
@@ -378,7 +378,8 @@ module.exports = {
 			},
 		},
 		{
-			files: ['./packages/nodes-base/nodes/**/*.ts'],
+			files: ['./packages/nodes-base/nodes/**/*.ts',
+			'./packages/nodes-base/credentials/*.credentials.ts'],
```

Comment:
Can you please add this cred rule in the override immediately above? And keep the path in this override only for nodes? Thanks!

Reply:
Fixed it